newton/_src/utils/update_usd.py

```
@@ -0,0 +1,510 @@
+        self.stage.SetTimeCodesPerSecond(self.fps)
+
+        for prim_path in self.path_body_map.keys():
+            prim = self.stage.GetPrimAtPath(Sdf.Path(prim_path))
+            self._xform_to_tqs(prim)
+
+    def _precompute_parents_xform_inverses(self):
+        """
+        Convention: prefix c is for **current** prim.
+        Prefix p is for **parent** prim.
+        """
+        from pxr import Sdf, Usd  # noqa: PLC0415
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)